src/c_api.rs

```diff
@@ -686,8 +684,6 @@ pub enum blazesym_sym_type {
 #[repr(C)]
 #[allow(non_camel_case_types, unused)]
 pub enum blazesym_faf_type {
-    /// Invalid type
-    BLAZESYM_FAF_T_INVALID,
```

Comment:
I'm a bit undecided on this. Judging by a lot of previous C-side experiences with kernel UAPIs, having 0 match to something invalid and requiring users to actively set non-zero values is proving pretty valuable in UAPI design.

Reply:
I agree that our variants should have non-zero values. I was planning on getting there when we address https://github.com/libbpf/blazesym/issues/84.
However, having an "invalid" variant makes no sense. All it forces us to do is have a two staged validation process: first we need to check whether we have a valid Rust `enum` and then we need to check whether it's invalid. It has no meaning, so this variant should not exist.

Reply:
alright